src/package-lock.json

```diff
@@ -1,1214 +1,8 @@
 {
   "name": "almanac.httparchive.org",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Can we revert these changes? We use npm v7 on this project so version 2 of the package-lock.json file. It's that that you are using an older version, but no need to change the package-lock file.